skbio/io/format/tests/test_embl.py

```diff
@@ -0,0 +1,773 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import io
+from unittest import TestCase, main
+
+import skbio.io
+
+from skbio import DNA, RNA, Sequence, Protein
+from skbio.metadata import IntervalMetadata
+from skbio.util import get_data_path
+
+# Module specific execption and functions
+from skbio.io import EMBLFormatError
+
+from skbio.io.format.embl import (
+    _embl_sniffer, _parse_id, _parse_reference, _embl_to_generator,
+    _get_embl_section, _embl_to_sequence, _embl_to_dna,
+    _embl_to_rna, _embl_to_protein,
+    _generator_to_embl, _sequence_to_embl,
+    _protein_to_embl, _rna_to_embl, _dna_to_embl,
+    _serialize_id, _parse_assembly)
+
+
+class SnifferTests(TestCase):
+    def setUp(self):
+        self.positive_fps = list(map(get_data_path, [
+            'embl_single_record',
+            'embl_multi_records']))
+
+        self.negative_fps = list(map(get_data_path, [
+            'empty',
+            'whitespace_only',
```

Comment:
would it be possible to add in an entry that has extra whitespace at the start of the file? in other words, it has valid embl content, but the file is malformed, which should trigger a `False` on parse